resources/route53_domains.go

```diff
@@ -0,0 +1,552 @@
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "abuse_contact_phone",
+				Description: "Phone number for reporting abuse.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "admin_privacy",
+				Description: "Specifies whether contact information is concealed from WHOIS queries",
+				Type:        schema.TypeBool,
+			},
+			{
+				Name:        "auto_renew",
+				Description: "Specifies whether the domain registration is set to renew automatically.",
+				Type:        schema.TypeBool,
+			},
+			{
+				Name:        "creation_date",
+				Description: "The date when the domain was created as found in the response to a WHOIS query.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "dns_sec",
+				Description: "Reserved for future use.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "expiration_date",
+				Description: "The date when the registration for the domain is set to expire",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "registrant_privacy",
+				Description: "Specifies whether contact information is concealed from WHOIS queries",
+				Type:        schema.TypeBool,
+			},
+			{
+				Name:        "registrar_name",
+				Description: "Name of the registrar of the domain as identified in the registry",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "registrar_url",
+				Description: "Web address of the registrar.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "registry_domain_id",
+				Description: "Reserved for future use.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "reseller",
+				Description: "Reseller of the domain",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "status_list",
+				Description: "An array of domain name status codes, also known as Extensible Provisioning Protocol (EPP) status codes",
+				Type:        schema.TypeStringArray,
+			},
+			{
+				Name:        "tech_privacy",
+				Description: "Specifies whether contact information is concealed from WHOIS queries",
+				Type:        schema.TypeBool,
+			},
+			{
+				Name:        "updated_date",
+				Description: "The last updated date of the domain as found in the response to a WHOIS query.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "who_is_server",
+				Description: "The fully qualified name of the WHOIS server that can answer the WHOIS query for the domain.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "tags",
+				Description: "A list of tags",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveRoute53DomainTags,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "aws_route53_domain_admin_contact_extra_params",
```

Comment:
this relation can be a json, where name is the key and value is the value

Reply:
Done